include/aspect/simulator_access.h

```diff
@@ -305,6 +305,15 @@ namespace aspect
       convert_output_to_years () const;
 
       /**
+       * Return whether or not we do the correction of
+       * the Stokes right hand side vector to ensure that the average
+       * divergence is zero. This is necessary for compressible models, but
+       * only if there are no in/outflow boundaries.
+       */
+      bool
+      do_pressure_rhs_compatibility_modification () const;
```

Comment:
A function that starts with a verb suggests that it *does* something, but this one doesn't. How about `pressure..._is_enabled()`?